### PR TITLE
Update URL params for notitication items to point to its configuratio…

### PR DESCRIPTION
**Pull request by radekkaluzik**
…n when clicked on Manage this event

https://issues.redhat.com/browse/RHCLOUD-29374

### CODE REVIEW

File: src/components/NotificationsDrawer/NotificationItem.tsx [hunk 1]
```diff
@@ -46,7 +46,10 @@ const NotificationItem: React.FC<NotificationItemProps> = ({ notification, onNav
 
   const notificationDropdownItems = [
     <DropdownItem key="read" onClick={onMarkAsRead}>{`Mark as ${!notification.read ? 'read' : 'unread'}`}</DropdownItem>,
-    <DropdownItem key="manage-event" onClick={() => onNavigateTo('settings/notifications/configure-events')}>
+    <DropdownItem
+      key="manage-event"
+      onClick={() => onNavigateTo(`settings/notifications/configure-events?bundle=${notification.bundle}&tab=configuration`)}
```
Comment (Hyperkid123) [inline src/components/NotificationsDrawer/NotificationItem.tsx:51]: The URL has to be absolute. If you click the item from a nested router, the pathname will append and you will end up on non existing page.

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]

File: src/components/NotificationsDrawer/NotificationItem.tsx (hunk 1 above)
Comment (radekkaluzik) [reply src/components/NotificationsDrawer/NotificationItem.tsx]: fixed

File: src/state/atoms/notificationDrawerAtom.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ export type NotificationData = {
   source: string;
   bundle: string;
   created: string;
+  bundle: string;
```
Comment (aneelac22) [inline src/state/atoms/notificationDrawerAtom.ts:12]: bundle already exists

**Review by aneelac22 — APPROVED** [review]

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**